system/common.xml

```diff
@@ -350,16 +350,19 @@
 				</copy>
 			</then>
 		</if>
-		<condition property="isSVTTestRepo" value="true">
-			<isset property="env.isSVTTestRepo"/>
-		</condition>
-		<condition property="isSVTTestRepo" value="false">
-   		 	<not>
-				<isset property="env.isSVTTestRepo"/>
-			</not>
+		<property name="IS_SVT_TESTREPO" value="${env.IS_SVT_TESTREPO}"/>
```

Comment:
Would removing this line affect the running? Seem the `IS_SVT_TESTREPO` is set in the following `condition` blocks either way

Reply:
Yes I have removed the line on setting the value to IS_SVT_TESTREPO. Tests below passed
https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/41978/ - without SVTTestRepo
https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder_CR/19368/ - with SVTTestRepo